src/include/gcc.h

```diff
@@ -280,6 +280,23 @@ WT_ATOMIC_FUNC(size, size_t, size_t *vp, size_t v)
         __asm__ volatile("" ::: "memory"); \
     } while (0)
 
+
+#elif defined(__riscv) && (__riscv_xlen == 64)
+#define WT_PAUSE() __asm__ volatile("nop" ::: "memory")
```

Comment:
There is a `pause` instruction in RISC-V but I don't think compilers support it yet: https://riscv.org/announcements/2021/02/risc-v-international-unveils-fast-track-architecture-extension-process-and-ratifies-zihintpause-extension/

It is interesting, because it is a `fence` instruction with a null successor set. I didn't want to gamble on writing it that way, so I just went with `nop` for now.

Reply:
I'd suggest a comment here, plus links?

Reply:
Added, will push an update soon.

Reply:
I can't get the assembler I have to accept any likely syntax, though. "fence w,0" is rejected and "fence w," assembles to "fence w,w". Blah :-(

Reply:
Thanks for trying it out though!